properties.py

```diff
@@ -1460,14 +1465,14 @@ class RendermanObjectSettings(bpy.types.PropertyGroup):
         default=False)
     motion_segments = IntProperty(
         name="Motion Samples",
-        description="Number of motion samples to take for multi-segment motion blur",
+        description="Number of motion samples to take for multi-segment motion blur.\nUse higher numbers for curved motion trails.",
         min=2, max=16, default=2)
 
     shadinginterpolation = EnumProperty(
         name="Shading Interpolation",
         description="Method of interpolating shade samples across micropolygons",
         items=[('constant', 'Constant', 'Flat shaded micropolygons'),
-               ('smooth', 'Smooth', 'Gourard shaded micropolygons')],
+               ('smooth', 'Smooth', 'Smooth Gourard shaded micropolygons')],
```

Comment:
Oh....I had noticed the 'export smooth normals' option had vanished from the mesh options and I thought this had replaced it :O